framework/base/helpers/dom.js

```diff
@@ -3,3 +3,21 @@ export const queryAll = (selector, context = document) =>
   [].slice.call(context.querySelectorAll(selector));
 
 export default queryAll;
+
+export const hasClass = (element, cssClass) =>
```

Comment:
well the two other functions could be removed too, but so you will have to update your code